io_scene_godot/__init__.py

```diff
@@ -75,43 +75,51 @@ class ExportGodot(bpy.types.Operator, ExportHelper):
     use_export_shape_key = BoolProperty(
         name="Export Shape Key",
         description="Export all the shape keys in mesh objects",
-        default=True,
+        default=False,
```

Comment:
Why this change?

Reply:
The shape key is usually time consuming to export, so I think it is better that user explicitly check it when they really want to export

Reply:
But if shapekeys are present in a scene, a user will probably want to export them.